src/actions/guides/tutorial/03_design.cr

```diff
@@ -0,0 +1,173 @@
+
+    > For more information on rendering HTML, read the [Rendering HTML](#{Guides::Frontend::RenderingHtml.path}) guide.
+
+    ## Updating the Layout
+
+    As mentioned previously, we currently have two separate layouts. One for when users are logged in `MainLayout`, and one for
+    when users are logged out `AuthLayout`. These files live in the root of your `src/pages/` directory. We will update our `AuthLayout`
+    to start so we can get a feel for writing HTML in Lucky.
+
+    When it comes to writing HTML in Lucky, we use plain Crystal methods that generate HTML for us!
+    [Read the HTML guide](#{Guides::Frontend::RenderingHtml.path}) for more details.
+
+    ### Anatomy of the layout
+
+    Open up the `AuthLayout` in `src/pages/auth_layout.cr`. This file has a `render` method which contains the start to our HTML. Within this method
+    are several other Crystal methods like `html_doctype`, `html`, and `body` which should be pretty recognizable. Any HTML tag has an associated method
+    you can call from here. In addition, there's a few helper methods which are explained more in the [HTML guide](#{Guides::Frontend::RenderingHtml.path}).
+
+    Two methods that may look unfamiliar are `mount`, and the `content` method you see inside of the `body` block.
+    The `mount` is related to [Components](#{Guides::Frontend::RenderingHtml.path(anchor: Guides::Frontend::RenderingHtml::ANCHOR_COMPONENTS)})
+    which we will cover later. The `content` method is used to render the page content from pages that inherit from this layout.
+    For example, our `Home::IndexPage` inherits from `AuthLayout`, so the `Home::IndexPage` has a `content` method where all of the Home page HTML will go.
+
+    ### Adding a wrapper
+
+    Let's update the `html` block, and replace it with this code:
+
+    ```crystal
+    # src/pages/auth_layout.cr
+    html class: "h-100", lang: "en" do
+      mount Shared::LayoutHead, page_title: page_title, context: context
+
+      body class: "d-flex flex-column h-100" do
+        mount Shared::FlashMessages, context.flash
+        main class: "flex-shrink-0" do
+          content
+        end
+
+        footer class: "footer mt-auto py-3 bg-light" do
+          div class: "container" do
+            span "CloverApp", class: "text-muted"
+          end
+        end
+      end
+    end
+    ```
+
+    As you can see, we can write methods that match up with HTML tags like `span`, `div`, `main`, and `footer`. We can
+    even add our own custom CSS classes by using the `class: ""` argument of each tag method.
+
+    Boot up your app, and give it a shot! See that you now have "CloverApp" displaying below the content of your Home page.
+
+    ## Adding a CSS framework
+
+    If you haven't guessed by now, we've started adding [Bootstrap](https://getbootstrap.com/) classes to our HTML. You're free
+    to use any (or no) CSS framework you wish. There's no limitations since Lucky includes [LaravelMix](https://laravel-mix.com/)
```

Comment:
```suggestion
    to use any (or no) CSS framework you wish. There are no limitations since Lucky includes [LaravelMix](https://laravel-mix.com/)
```